build.gradle

```diff
@@ -25,20 +24,6 @@ allprojects {
     }
 }
 
-// TODO Remove once either Facebook or Robolectric fix the crash: https://github.com/robolectric/robolectric/issues/2579#issuecomment-276526911
-gradle.taskGraph.whenReady { graph ->
```

Comment:
Nice!  Isn't there some hack we put in the manifest we can remove now too?  Or was that unrelated.